ci/prow/boskos/set_permissions.sh

```diff
@@ -65,3 +68,14 @@ for res in ${RESOURCES[@]}; do
   echo "NOTE: Adding ${res} as ${role}"
   gcloud projects add-iam-policy-binding ${PROJECT} --member ${type}:${res} --role ${role}
 done
+
+# As required by step 6 in https://github.com/google/knative-gcp/tree/master/docs/storage,
+# grant the GCS service account the permissions to publish to GCP Pub/Sub.
+echo "Activating GCS service account"
+curl -s -X GET -H "Authorization: Bearer $(gcloud auth application-default print-access-token)" "https://www.googleapis.com/storage/v1/projects/${PROJECT}/serviceAccount"
```

Comment:
I think it would be nice having the access token as a read-only var at the beginning of the script, as well as the project number. As a bonus, if there's something wrong the script will fail fast, before any changes are made.

Reply:
Done.